corehq/apps/app_manager/views/translations.py

```diff
@@ -86,7 +89,10 @@ def download_bulk_app_translations(request, domain, app_id):
     temp = io.BytesIO()
     data = [(k, v) for k, v in six.iteritems(rows)]
     export_raw(headers, data, temp)
-    return export_response(temp, Format.XLS_2007, "bulk_app_translations")
+    filename = '{app_name} v.{app_version} - App Translations'.format(
+        app_name=app.name,
+        app_version=app.version)
+    return export_response(temp, Format.XLS_2007, filename)
```

Comment:
@orangejenny used this couple of times recently for some work on translations. 
This change definitely makes it better.
Does it add value to add a timestamp to it as well to differentiate files downloaded on the same day?
Which is already possible by checking the file's timestamp but just makes it easier/explicit (feel free to ignore this comment if does not sound useful)

Reply:
Hey, I agree it's a decent idea, but with it being available as the file's date created, I'm inclined to leave it out of the filename.